app.js

```diff
@@ -34,7 +34,7 @@ app.post(
 
 app.get('/status', (req, res) => res.send('contacts is alive'))
 
-app.use(function(error, req, res, next) {
+app.use(function (error, req, res, next) {
```

Comment:
I guess Prettier is now adding whitespace before params for callback functions only